source/kmip.rst

```diff
@@ -0,0 +1,86 @@
+.. _kmip:
+
+Using the Key Management Interoperability Protocol (KMIP) 
+============================================================
+
+This feature is **technical preview** quality.
+
+|PSMDB| adds support for secure transfer of keys using the `OASIS Key Management Interoperability Protocol (KMIP) <https://docs.oasis-open.org/kmip/kmip-spec/v2.0/os/kmip-spec-v2.0-os.html>`__. The KMIP implementation was tested with the `PyKMIP server <https://pykmip.readthedocs.io/en/latest/server.html>`__ and the `HashiCorp Vault Enterprise KMIP Secrets Engine <https://www.vaultproject.io/docs/secrets/kmip>`__.
+
+KMIP enables the communication between key management systems and the database server. KMIP can perform the following operations:
+
+* Streamline encryption key management
+* Eliminate redundant key management processes
+
+.. admonition:: KMIP parameters
+
+   .. list-table::
+      :widths: 25 25 15 35
+      :header-rows: 1
+   
+      * - Option
+        - Type
+        - Description
+      * - --kmipServerName
+        - string
+        - The hostname or IP address of the KMIP server. You can specify multiple KMIP servers as a comma-separated list, for example, *server1.example.com,server2.example.com*. On startup, MongoDB will select the first server in the order list, to which it can successfully establish a connection.
+      * - --kmipPort
+        - number
+        - The port used to communicate with the KMIP server.
+      * - --kmipRotateMasterKey
+        - switch
+        - Enables master key rotation.    
+      * - --kmipClientCertificateFile
+        - string
+        - The path to the client certificate file. The client certificate file is used by MongoDB to authenticate to the KMIP server. On macOS or Windows, you can use a certificate from the operating system's secure store instead of a PEM key file. Use ``--kmipClientCertificateSelector`` for this purpose.
```

Comment:
"On macOS or Windows, you can use a certificate from the operating system's secure store instead of a PEM key file." - does psmdb support windows and macos at all?

Reply:
Removed